pkg/scheduler/apis/config/v1alpha1/zz_generated.conversion.go

```diff
@@ -110,12 +108,14 @@ func autoConvert_v1alpha1_KubeSchedulerConfiguration_To_config_KubeSchedulerConf
 	if err := Convert_v1alpha1_KubeSchedulerLeaderElectionConfiguration_To_config_KubeSchedulerLeaderElectionConfiguration(&in.LeaderElection, &out.LeaderElection, s); err != nil {
 		return err
 	}
-	if err := configv1alpha1.Convert_v1alpha1_ClientConnectionConfiguration_To_config_ClientConnectionConfiguration(&in.ClientConnection, &out.ClientConnection, s); err != nil {
+	// TODO: Inefficient conversion - can we improve it?
+	if err := s.Convert(&in.ClientConnection, &out.ClientConnection, 0); err != nil {
```

Comment:
this conversion isn't right

Reply:
ugh, I have no idea what happened there wrt the autogenerated code. Can't reproduce either

Reply:
I have seen this independently from this PR. Looks like we have another code generation bug, maybe in the vicinity of Makefile.generated_files. /cc @thockin